### PR TITLE
Cleanup documentation

### PR DESCRIPTION
**Pull request by RachiLam**
Update links and documented npm scripts. There were some documented npm scripts that didn't look right but I didn't know how to update them: `npm run cover`, `npm run cover:up`, and `npm run dist`.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -12,20 +12,20 @@ If you are submitting a bug report, please include the following:
...
 * When submitting *Pull Requests (PRs)*, you should request to merge your changes into our `develop` branch.
-* Our approach to testing is described in [spec/README.md](/spec/README.md).
+* Our approach to testing is described in [docs/TESTING.md](/docs/TESTING.md).
 * PRs without specs will not be merged anytime soon!
 
 ## Releasing new versions
 
 * Create a new entry in the [CHANGELOG](/CHANGELOG) file,
-* Re-build and re-release the online demo, see [`website-jsf` gh-pages branch](https://github.com/json-schema-faker/website-jsf/tree/gh-pages).
+* Re-build and re-release the online demo, see [`json-schema-faker` gh-pages branch](https://github.com/json-schema-faker/json-schema-faker/tree/gh-pages).
```
Comment (RachiLam) [inline .github/CONTRIBUTING.md:21]: It looks like the website was merged into this repository so I believe this change is correct.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -12,20 +12,20 @@ If you are submitting a bug report, please include the following:
...
 
 ## Development tasks
 
 * `npm run dev` &mdash; Run the tests and watch (preferred during development)
-* `npm run dev:lint` &mdash; Run eslint on all sources
-* `npm run dev:unit` &mdash; Run `jasmine-node` on unit tests
-* `npm run dev:schema` &mdash; Run `jasmine-node` on schema tests
+* `npm run lint` &mdash; Run eslint on all sources
+* `npm run test:unit` &mdash; Run unit tests
+* `npm run test:schema` &mdash; Run schema tests
```
Comment (RachiLam) [inline .github/CONTRIBUTING.md:28]: I would like to update the other npm scripts here, but idk what they should be.

File: docs/USAGE.md [hunk 3]
```diff
@@ -350,7 +349,7 @@ jsf.option('alwaysFakeOptionals', true);
 
 ## Extending dependencies
 
-You may extend [Faker.js](http://marak.com/faker.js/):
+You may extend [Faker.js](https://github.com/Marak/faker.js):
```
Comment (RachiLam) [inline docs/USAGE.md:352]: I couldn't get to `http://marak.com/faker.js/`

File: docs/USAGE.md [hunk 4]
```diff
@@ -4,7 +4,6 @@
   - [Overview](#overview)
   - [Example usage](#example-usage)
     - [More examples](#more-examples)
-    - [Gist demos](#gist-demos)
```
Comment (RachiLam) [inline docs/USAGE.md:7]: I couldn't find this section

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (pateketrueke) [reply .github/CONTRIBUTING.md:21]: That's true, rebuilding the demo is not useful until you want to add/fix stuff there (and only if it's related/needed).

We should make such step optional, back to that time we probably wanted to include a built version on the website (not anymore thanks to unpkg).

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (pateketrueke) [reply .github/CONTRIBUTING.md:28]: This is fine, I'll update those on time once I remove those we no longer need.

File: docs/USAGE.md (hunk 3 above)
Comment (pateketrueke) [reply docs/USAGE.md:352]: Thank you!

File: docs/USAGE.md (hunk 4 above)
Comment (pateketrueke) [reply docs/USAGE.md:7]: Yeah, we wanted to list here those gists we were using on the webapp for reference purposes.

**Pull request merged**